src/GitVersion.BuildAgents.Tests/Agents/BitBucketPipelinesTests.cs

```diff
@@ -161,5 +161,14 @@ private void AssertVariablesAreWrittenToFile(string file)
         props.ShouldContain("export GITVERSION_MINOR=2");
         props.ShouldContain("export GITVERSION_SHA=f28807e615e9f06aec8a33c87780374e0c1f6fb8");
         props.ShouldContain("export GITVERSION_COMMITDATE=2022-04-06");
+
+        File.Exists("gitversion.ps1").ShouldBe(true);
```

Comment:
pass the `powershell` file as method parameter similar to the property file.
Please rename the `file` parameter to `propertyFile` to make it clear which file it is

Reply:
Fixed.